ci/build_container/build_container.sh

```diff
@@ -32,9 +26,10 @@ pip install virtualenv
 export GOPATH=/usr/lib/go
 go get github.com/bazelbuild/buildifier/buildifier
 
-# GCC 4.9 for everything.
-export CC=gcc-4.9
-export CXX=g++-4.9
+# GCC for everything.
+export CC=gcc
+export CXX=g++
+g++ --version
```

Comment:
Just compare the exact string - this is only used in the container where that will be stable.